builtin/stash--helper.c

```diff
@@ -694,6 +700,43 @@ static int show_stash(int argc, const char **argv, const char *prefix)
 	return ret;
 }
 
+static int store_stash(int argc, const char **argv, const char *prefix)
+{
+	struct object_id obj;
+	int ret = 0;
+	const char *stash_msg = NULL;
+	const char *w_commit = NULL;
+	struct option options[] = {
+		OPT__QUIET(&quiet, N_("be quiet, only report errors")),
+		OPT_STRING('m', "message", &stash_msg, "message", N_("stash message")),
+		OPT_END()
+	};
+
+	argc = parse_options(argc, argv, prefix, options,
+		git_stash_helper_store_usage, PARSE_OPT_KEEP_UNKNOWN);
+
+	if (argc == 0) {
+		fprintf(stderr, _("\"git stash--helper store\" requires one <commit> argument\n"));
+		return -1;
+	}
+
+	w_commit = argv[0];
+
+	if (!stash_msg)
+		stash_msg  = xstrdup("Created via \"git stash--helper store\".");
+
+	ret = get_oid(w_commit, &obj);
+	if (!ret) {
+		ret = update_ref(stash_msg, ref_stash, &obj, NULL,
+			REF_FORCE_CREATE_REFLOG, 0);
```

Comment:
I wonder whether we really want to pass `0` here instead of `UPDATE_REFS_MSG_ON_ERR`. I think we should pass the latter. Or maybe even better `quiet ? UPDATE_REFS_QUIET_ON_ERR : UPDATE_REFS_MSG_ON_ERR`.